cmd/cosign/cli/fulcio/fulcio.go

```diff
@@ -92,6 +99,14 @@ func GetCert(ctx context.Context, priv *ecdsa.PrivateKey, idToken, flow, oidcIss
 		c.flow = oauthflow.NewDeviceFlowTokenGetter(
 			oidcIssuer, oauthflow.SigstoreDeviceURL, oauthflow.SigstoreTokenURL)
 	case FlowNormal:
+		fmt.Fprintln(os.Stderr, PrivacyStatement)
```

Comment:
Why not pass `PrivacyStatement` to `ConfirmPrompt`?

Reply:
ConfirmPrompt printed the message after the confirmation check, and I wanted to print it before

Reply:
Unless I'm misunderstanding your claim, it doesn't seem to: 

https://github.com/sigstore/cosign/blob/03e66aad02cf7c987ea489cdeda1fd580b6b1fc6/pkg/cosign/common.go#L38-L39

I also submit that if `ConfirmPrompt` doesn't work for this use case directly, there's something wrong with it and we should fix it.

Reply:
%s was invisible to me 🙃

Reply:
Fixed